.github/workflows/ci.yml

```diff
@@ -0,0 +1,92 @@
+name: Run Tests
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
```

Comment:
`  workflow_dispatch:` is a handy trigger as well, as it allows you to run a build arbitrarily without needing to push code.

Reply:
Added!